docs/admin/auth/hba.rst

```diff
@@ -200,46 +200,3 @@ For more information, consult the :ref:`privileges section
 
     cr> DROP USER admin;
     DROP OK, 1 row affected (... sec)
-
-
-.. _admin_hba_node:
-
-Node-to-node communication
-==========================
```

Comment:
I think we should still keep the documentation but remove the note about ssl-mixed note.
It is still important to activate the client certificate check if you're communicating across trusted network borders.

Reply:
restored this section, removed only sentences about dual mode and ssl-mixed